### ISSUE TITLE
Battery questions

### ISSUE THREAD

**Issue by apexad** [iAnatoly/homebridge-aladdin-connect-garage-door#7]
Let me know if you actually get a low battery warning. In my case, it never happened and pretty sure my battery just died.

Curiously, it appears when there is no contact with the door position sensor, battery reports as 100%.

**Comment by apexad** [comment]
Logged in at https://genie.exosite.com I see there is another value form the API `dps#.link_status` that I think could be of use... My research shows it is most likely the following:

1 - Not Paired
2 - Paired, but not connected
3 - Paired, connected

If this is accurate, I think we would just need to update https://github.com/apexad/node-aladdin-connect-garage-door to also poll this value along with battery_level and just report battery as 0 if this value is 2.  What do you think?

**Comment by iAnatoly** [comment]
Hi, sorry fir the delay - it was a crazy week at work. Answering your question - I actually have set the warning level at 85% or something, and I did receive a warning after I opened/closed the door. So in theory, this should not be an issue if you catch the warning badge at 20%. 

On the other hand, since apple home does not poll the battery in background, it is theoretically possible to miss the warning (especially if outside temp is low, and battery dies quickly), so using "connection lost - battery is likely dead" approach should work. 

How can I help with confirming the values?

**Comment by apexad** [comment]
So, I think I got this right... I've published `node-aladdin-connect-garage-door@0.1.2` now with the fix.
Since minor was bumped a few versions back and you updated this plugin already to pull in 
```json
"node-aladdin-connect-garage-door": "^0.1.1"
````
...nothing further has to be done to pick up the change.  but for existing users, this plugin will beed to be re-installed to pickup the new node library.

Up to you if you want to bump this dependency and push a new release of this plugin.
